### DISCUSSION TITLE
`自定义顶栏` 的 `排行` 加入 `全站音乐榜` 和 `短剧榜` 入口

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#4101] [功能建议 / Ideas]
https://github.com/the1812/Bilibili-Evolved/issues/3763#issuecomment-1287810939 提过加入 `全站音乐榜` 入口。

`自定义顶栏` 的 `排行` 加入 `全站音乐榜` 和 `短剧榜` 入口。
（新加的这个 `短剧榜` 还收了 Vlog 我是没看懂的）
![image](https://user-images.githubusercontent.com/49396421/227786271-f5673ecd-4008-450b-9801-2a7b89b25e82.png)

顺便提醒还得适配一下夜间模式。

**Comment by lovelyCARDINAL** [comment]
#4154 完成后等一个feat: completed 的标签
